src/main/resources/assets/seedqueue/lang/pl_pl.json

```diff
@@ -0,0 +1,123 @@
+  "seedqueue.menu.benchmark.result": "%s resetów w %ss (%s RPS)",
+
+  "seedqueue.menu.config.useWall.notAvailable": "§c§oNie Dostępne",
+  "seedqueue.menu.config.useWall.notAvailable.tooltip": "Używanie Ekranu Ściany potrzebuje zainstalowania tych modów::",
+  "seedqueue.menu.config.useWall.notAvailable.tooltip.1": " - Sodium",
+  "seedqueue.menu.config.useWall.notAvailable.tooltip.2": " - StandardSettings",
+  "seedqueue.menu.config.useWall.notAvailable.tooltip.3": " - WorldPreview",
+
+  "seedqueue.menu.config.showAdvancedSettings.confirm.title": "Czy na pewno chcesz włączyć Ustawienia Zaawansowane?",
+  "seedqueue.menu.config.showAdvancedSettings.confirm.message": "Używaj tych opcji tylko jeśli jesteś pewien co robią albo gdy jesteś informowany przez kogoś kto wie co robi!",
+
+  "seedqueue.menu.keys": "Skróty Klawiszowe Ściany",
+  "seedqueue.menu.keys.configure": "Konfiguruj...",
+  "seedqueue.menu.keys.primary": "Klawisz Główny",
+  "seedqueue.menu.keys.secondary": "§7§oKlawisze Pomocnicze",
+  "seedqueue.menu.keys.secondary.tooltip": "Kiedy zostanie naciśnięty klawisz główny, gra sprawdzi, czy wszystkie klawisze pomocnicze są również naciśnięte, zanim aktywuje się skrót klawiaturowy.",
+  "seedqueue.menu.keys.blocking": "§7§oKlawisze Blokujące",
+  "seedqueue.menu.keys.blocking.tooltip": "Kiedy zostanie naciśnięty klawisz główny, gra sprawdzi, czy żadne klawisze blokujące §lnie§r są naciśnięte, zanim aktywuje się skrót klawiaturowy.",
+  "seedqueue.menu.keys.secondary_list": "§7§oPomocniczy:",
+  "seedqueue.menu.keys.blocking_list": "§7§oBlokujący:",
+
+  "seedqueue.key.categories.builtin": "Wbudowane Skróty Klawiszowe",
+  "seedqueue.key.categories.custom": "Niestandardowe Skróty Klawiszowe",
+  "seedqueue.key.play": "Zagraj w ten Świat",
+  "seedqueue.key.lock": "Użyj Kłódki na Świat",
+  "seedqueue.key.reset": "Zresetuj Świat",
+  "seedqueue.key.resetAll": "Zresetuj wszystkie Światy",
+  "seedqueue.key.focusReset": "Skupiony Reset",
+  "seedqueue.key.resetRow": "Zresetuj Rząd",
+  "seedqueue.key.resetColumn": "Zresetuj Kolumnę",
+  "seedqueue.key.playNextLock": "Zagraj w Następną Kłódkę",
+  "seedqueue.key.startBenchmark": "Zacznij Benchmark",
+  "seedqueue.key.cancelBenchmark": "Skończ Benchmark",
+
+  "speedrunapi.config.seedqueue.category.queue": "Ustawnienia Kolejki",
+  "speedrunapi.config.seedqueue.category.chunkmap": "Ustawienia Chunkmap",
+  "speedrunapi.config.seedqueue.category.wall": "Ustawienia Ekranu Ściany",
+  "speedrunapi.config.seedqueue.category.performance": "Ustawnienia Wydajności",
+  "speedrunapi.config.seedqueue.category.advanced": "Ustawienia Zaawansowane",
+  "speedrunapi.config.seedqueue.category.threading": "Ustawienia Wątków",
+  "speedrunapi.config.seedqueue.category.experimental": "Ustawienia Eksperymentalne",
+  "speedrunapi.config.seedqueue.category.debug": "Ustawienia Debugujące",
+
+  "speedrunapi.config.seedqueue.option.maxCapacity": "Maksymalna Ilość Seedów w Kolejce",
```

Comment:
Minecraft tłumaczy na `ziarno`, ale jak nie tłumaczymy `SeedQueue` to można zostać przy tym